src/dynarec/la64/dynarec_la64_660f.c

```diff
@@ -724,10 +800,43 @@ uintptr_t dynarec64_660F(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int
                         VFRINT_D(v1, q1);
                         x87_restoreround(dyn, ninst, u8);
                     } else {
-                        VFRINTRRD_D(v1, q1, round_round[u8 & 3]);
+                        VFRINTRRD(v1, q1, round_round_D[u8 & 3]);
                     }
                     VEXTRINS_D(q0, v1, 0);
                     break;
+                case 0x0C:
+                    INST_NAME("BLENDPS Gx, Ex, Ib");
+                    nextop = F8;
+                    GETGX(q0, 1);
+                    GETEXSS(q1, 0, 1);
+                    u8 = F8&0b1111;
+                    if(u8==0b0011) {
```

Comment:
I don't understand the `==` here, the implementation is correct, but wouldn't it be better if we change `u8 == 0b0011` to `(u8 & 0b11) == 0b11`, bitto below?

... okay I saw this is copied from the Arm side; probably applies to Arm too? @ptitSeb

Reply:
Ah, yes, correct indeed. I'll fix the Arm64 side!